Mbarak-Mbigo/cp2-frontend
# Ft login logic

#### What does this PR do?
Fix bucket subject bug
#### Description of Task to be completed?
Fix bucket subject bug
#### How should this be manually tested?
Start application and login:
Expected outcome: Application should work okay in login, creation of buckets and view bucket items
#### Any background context you want to provide?
N/A
